admin/sales_analytics/metrics.py

```diff
@@ -0,0 +1,113 @@
+from admin.base.settings import ENTRY_POINTS
+from admin.sales_analytics.utils import get_entry_point, get_sorted_index
+from datetime import datetime, timedelta
+from framework.mongo import database as db
+
+
+def get_user_count(db=db, entry_points=ENTRY_POINTS):
+    """
+    Get the number of users created from each entry point: osf, osf4m, prereg, and institution.
+    """
+    count_list = []
+    percent_list = []
+    tags = ['osf4m', 'prereg', 'institution', 'osf']
+    total = db.user.find({}).count()
+    for i in entry_points:
+        count = db.user.find({'system_tags': i}).count()
+        percent = round(float(count) / float(total), 2)
+        count_list.append(count)
+        percent_list.append(percent)
+    osf_count = total - sum(count_list)
+    osf_percent = 1 - sum(percent_list)
+    count_list.append(osf_count)
+    percent_list.append(osf_percent)
+    sorted_index = get_sorted_index(count_list)
```

Comment:
Also it might be better to sort the list initially before using it in the for loop. Then you might not have to go through all of this post loop code.